inc/storefront-template-functions.php

```diff
@@ -159,14 +159,20 @@ function storefront_header_widget_region() {
 	 * @return void
 	 */
 	function storefront_site_branding() {
-		if ( function_exists( 'the_custom_logo' ) && has_custom_logo() ) {
-			the_custom_logo();
+    if ( function_exists( 'the_custom_logo' ) && has_custom_logo() ) {
+      $logo = get_custom_logo();
+
+      echo $logo = is_home() ? '<h1>' . $logo . '</h1>' : $logo;
```

Comment:
Is it definitely worth wrapping an image in a `h1`? 

Like you said this is gonna mess with the designer extension and probably all of the child themes as well. That's not ideal xD